src/jit/codegenarmarch.cpp

```diff
@@ -343,7 +343,11 @@ void CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
         case GT_LABEL:
             genPendingCallLabel       = genCreateTempLabel();
             treeNode->gtLabel.gtLabBB = genPendingCallLabel;
+#if defined(_TARGET_ARM)
+            emit->emitIns_J_R(INS_adr, EA_PTRSIZE, genPendingCallLabel, treeNode->gtRegNum);
```

Comment:
And generally speaking, as this usage of `emitIns_J_R()` was copied from `codegenlegacy.cpp`, may it be worth to refactor arm32 `emitIns_R_L()` instead? Though I'm not sure what is the difference intended between the `J_R` and `R_L` forms.